cellpack/autopack/ingredient/Ingredient.py

```diff
@@ -2975,16 +2978,14 @@ def attempt_to_pack_at_grid_location(
             )
 
         # grow doesnt use panda.......but could use all the geom produce by the grow as rb
-        if self.placeType == "jitter" or self.Type == "Grow" or self.Type == "Actine":
+        if self.Type == "Grow" or self.Type == "Actine":
```

Comment:
did you mean to remove the placeType check here? If placeType is jitter, we should call jitter_place

Reply:
the growing ingredient (Grow.py)  is a special ingredient that use its own placing function that deals with the placing type. To avoid confusion I renamed it to grow_place, while before it was overwriting the jitter_place function of the parent class Ingredient.

Reply:
I haven't pulled this down, but from the diff on here, it now looks like there is no way to call `jitter_place`. I think it makes more sense to have this just be (pseudocode) `if self.place == "jitter" call self.jitter_place` and then each ingredient has its own implementation of jitter (defaults to spheres). That way we can get rid of the `if self.Type == "Grow" or self.Type == "Actine"` check here completely

Reply:
To make it clear I separate the grow ingredient and the rest, since the grow ingredient deal itself with the different type methods.